File: sgilt-front/app/data/evenement/dto/EventCountsDto.ts
/**
 * DTO — contrat de réponse de l'API GET /events/:id/counts
 */
export interface EventCountsDto {
  mood: 'confirmee' | 'en_discussion' | 'nouvelle' | 'defaut'
  confirmedCount: number
  inDiscussionCount: number
  nouvelleCount: number
  refuseeCount: number
  annuleeCount: number
  realiseeCount: number
}

File: sgilt-core/src/main/java/net/franzka/sgilt/core/evenement/dto/EventCountsDto.java
package net.franzka.sgilt.core.evenement.dto;

public record EventCountsDto(
        String mood,
        int confirmedCount,
        int inDiscussionCount,
        int nouvelleCount,
        int refuseeCount,
        int annuleeCount,
        int realiseeCount
) {}
